src/data/constants/assets.js

```diff
@@ -0,0 +1,3 @@
+/* ASSETS RESPONSE MESSAGES */
+export const ERROR_FILE_DELETE = 'File could not be deleted.';
+export const SUCCESS_FILE_DELETE = 'File was successfully deleted.';
```

Comment:
Do we want the messages for responses to be more descriptive? Do we also want to store other static messages (i.e modal body text / confirmation button text / status alert text in this constants file as well, or do we prefer to have that inline? Is any of it reusable on other pages that we may do in the future?

Reply:
Also this file was created because of the use of checking the status message to determine if the request being returned by the action creator was a success or failure, moving the constant string to a shared location reduces the number of places that will need to get changed in the future if we decide to change it. This checking of the message currently determines where focus should move from the StatusAlert.

Reply:
We're going to need to consider i18n with these strings as well - the "inline vs centralized" string storage decision will need to be made and standardized across the repo soon.